### ISSUE TITLE
Brain areas - should multi-modal areas be "part of" multiple modalities?

### ISSUE THREAD

**Issue by stevevanhooser** [obophenotype/uberon#2411]
A general question in preparing new brain areas:

There are areas in the brain that provide functionality for more than one sensory system. Should these areas be both “part of”, say, visual cortex and “part of” somatosensory cortex? We could not find an example of multi-modal areas being double classified in Uberon, but in cases such as these it would seem correct to do so.

As an example, the posterior parietal rostral cortical area (PPr) of the ferret contains both a representation of the visual field and somatotopic representation of the contralateral body (Manger, Masiello & Innocenti, 2002; Dell, Innocenti, Hilgetag, & Manger, 2019).

(This is also buried in #2352 but I am pulling out this small issue so it is easier to nail down.)

Thanks
Steve

**Comment by stevevanhooser** [comment]
Adding @shawntanzk

**Comment by shawntanzk** [comment]
if an area is "part of" something, it must be true for the whole area - therefore something being "part of" two areas is probably problematic. Overlaps should probably be used instead. 

Do note that the above is representing spatial relations - if it overlaps those two areas, the assumption is that the part spatially overlaps what is known as the visual and somatosensory cortex in the ferret. If however you are looking for sensory capabilities, then 'capable of' might be a more appropriate relation: eg olfactory bulb: 'capable of' some 'sensory perception of smell'

hope that helps :)

**Comment by shawntanzk** [comment]
> (This is also buried in https://github.com/obophenotype/uberon/issues/2352 but I am pulling out this small issue so it is easier to nail down.)

sorry for not getting to this faster - will try to get through it sometime soon, but breaking down to questions like this is super helpful for me, thanks heaps!

**Comment by stevevanhooser** [comment]
Thanks for your messages. I think we are getting somewhere on this topic.

I guess it comes down to what one means by "visual cortex". The notion that I am most familiar with is expressed in this [University of Washington web page](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=1911&questID=1934): 

"The term visual cortex (rodent) refers to a set of cytoarchitecturally defined areas of the[ cerebral cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=39&questID=1911) of the[ mouse](http://braininfo.rprc.washington.edu/Species.aspx?ID=2540&questID=1911) and the[ rat](http://braininfo.rprc.washington.edu/Species.aspx?ID=2517&questID=1911) that are segmented differently by the authors of different atlases. Each system includes a[ primary visual area](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=2678&questID=1911), which corresponds to the[ primary visual cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=2102&questID=1911) in the[ primate](http://braininfo.rprc.washington.edu/Species.aspx?ID=2516&questID=1911), and a variable number of secondary areas."

To make it more generic, I'd say

"The term visual cortex refers to a set of cytoarchitecturally defined areas of the[ cerebral cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=39&questID=1911) that exhibit responses to retinal stimulation and that are segmented differently by the authors of different studies. Each system includes a[ primary visual area](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=2678&questID=1911), which corresponds to the[ primary visual cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=2102&questID=1911), and a variable number of secondary areas."

In this definition, an area would not have to be exclusively visual to be part of "visual cortex". Indeed, we are learning about non-visual functions of areas that are described in "visual cortex" of the mouse every year. Further, in many species, areas have been identified as having visual responses but the overall function or function(s) of the areas are not fully known.

In other words, I would not expect "auditory cortex" and "visual cortex" to be non-overlapping.

**Comment by stevevanhooser** [comment]
Also, maybe these are already in the ontology, but I find the relation

'capable of' some 'sensory perception of smell'

problematic..the animal is capable of sensory perception of smell, but to say that a brain area is capable of perception sounds weird to me. A brain area can exhibit responses to stimuli that correlate with perception but to say that the area is capable of it seems like an impossible leap. (I don't want to get side tracked! I guess I am suggesting something like "exhibits responses to visual stimulation" as something less ambiguous.)

**Comment by shawntanzk** [comment]
> Also, maybe these are already in the ontology, but I find the relation
> 
> 'capable of' some 'sensory perception of smell'
> 
> problematic...

This is an imported GO term - the definition does technically allow for this imo: 
"The series of events required for an organism to receive an olfactory stimulus, convert it to a molecular signal, and recognize and characterize the signal." 
But yeah its not consistently applied and I totally understand your point. But yes, let's not get sidetracked - as long as it isn't wrong, I think we can leave it for now :)

**Comment by stevevanhooser** [comment]
I'm starting to have a stronger opinion about the definition of "primary visual cortex" and "visual cortex", for your consideration.

The current definition of "primary visual cortex", UBERON_0002436 is

"A subdivision of the cytoarchitecturally defined occipital region of cerebral cortex in the human. Defined by the band of Gennari, which gives it the name striate (furrowed) area, it occupies the banks of the calcarine sulcus which are located in the cuneus and the lingual gyrus of the occipital lobe. Cytoarchitecturally it is bounded by the area 18 of Brodmann (human) which surrounds it ( Brodmann-1909 ). In the mouse ( Paxinos-2001 ) and the rat ( Swanson-1998 ) it is located on the dorsolateral surface of the cerebral hemisphere[BrainInfo]. [ BrainInfo:2102 ]"

This is a very human and primate-centric definition, with some acknowledgement that it exists in rats and mice. In a paper in 2007, when I was really digging into primary visual cortex across mammals, I wrote the following as a description of primary visual cortex, which I think works better across species. The analysis is mainly due to Leah Krubitzer (cited twice here).

"Primary visual cortex, variously called V1, striate cortex, or area 17, is found in all mammals (Krubitzer and Kaas 2005), from highly visual primates to subterranean animals with subcutaneous eyes such as the blind mole rat (Cooper and others 1993). V1 can be uniquely identified as a region in the occipital portion of the neocortex that 1) has a retinotopic map of space, 2) receives a major projection from the lateral geniculate nucleus, and 3) possesses a highly granulated layer 4 (Rosa and Krubitzer 1999). On the basis of these multiple similarities, it is widely believed that V1 is homologous across mammals." (Van Hooser, 2007)

Citations are

- Krubitzer L, Kaas J. 2005. The evolution of the neocortex in mammals: how is phenotypic diversity generated? Curr Opin Neurobiol 15:444-53.
- Cooper HM, Herbin M, Nevo E. 1993. Visual system of a naturally microphthalmic mammal: the blind mole rat, Spalax ehrenbergi. J Comp Neurol 328:313-50
- Rosa MG, Krubitzer LA. 1999. The evolution of visual cortex: where is V2? Trends Neurosci 22:242-8
- Van Hooser SD 2007. Similarity and diversity in visual cortex: is there a unifying theory of cortical computation? Neuroscientist 13(6):639-56.

**Comment by stevevanhooser** [comment]
...continuing, I think a definition of "visual cortex" (UBERON 

which is now 

"The part of the cerebral cortex responsible for processing visual information."

should be something like:


"The term visual cortex refers to a region of the[ cerebral cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=39&questID=1911) that exhibits responses to retinal stimulation and that are segmented differently by the authors of different studies. In each species, visual cortex includes the [ primary visual cortex](http://braininfo.rprc.washington.edu/CentralDirectory.aspx?ID=2102&questID=1911), and a variable number of additional areas that have been noted in species-specific studies."

Almost all areas are, with enough investigation, multi-modal, and I think this definition encompasses what many investigators have intended when they have defined "visual cortex" in a variety of studies of many mammals. The areas vary considerably across mammals (except for primary visual cortex), and the functions of most of these areas are incompletely understood.

Best
Steve
